examples/bri-3/src/bri/auth/api/auth.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Get, Post, Request } from '@nestjs/common';
+import { Public } from 'src/bri/decorators/public-endpoint';
+import { AuthService } from '../services/auth.service';
+import { generateMnemonic } from 'bip39';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Public()
+  @Post('login')
+  async login(@Request() req) {
+    return this.authService.login(req.body);
```

Comment:
agree ... please, stick to the agreed-upon pattern @Manik-Jain, we need to be 100% constant throughout the implementation otherwise we are building in maintenance nightmares when it comes to trouble shooting, upgrades etc.

Reply:
Right, this was a miss on my end. This will be pushed in the next commit